src/main/java/org/spongepowered/api/data/key/Keys.java

```diff
@@ -543,6 +543,14 @@
 
     public static final Key<Value<Vector3d>> LEFT_ARM_ROTATION = KeyFactory.fake("LEFT_ARM_ROTATION");
 
+    /**
+     * Represents the {@link Key} for representing the dominant {@link HandType}
+     * of a {@link Living} entity.
```

Comment:
@kashike You can't have support for getting and no support for setting, that's not how data manipulators work. It's either they're supported for all `Living` with corner cases that are clearly defined in the javadoc, or, they're supported only by `Agent`s.

Considering that `ArmorStand`s are extending `Living` and not `Agent`s, this should target `Living`, but *clearly* state that you cannot support `Player`s, and instead use the handed property.